engines/wintermute/ad/ad_scene.cpp

```diff
@@ -1218,6 +1363,8 @@ bool AdScene::compareObjs(const AdObject *obj1, const AdObject *obj2) {
 
 //////////////////////////////////////////////////////////////////////////
 bool AdScene::displayRegionContentOld(AdRegion *region) {
+	// is this function actually used?
```

Comment:
Did you grep for any callers?

Reply:
I used "Find references" in QtCreator, but grep is doesn't give anything back either apart from the definition itself. Anyways this was mainly a comment about whether some code has to be brought back form WME3D